src/components/tags/tags.scss

```diff
@@ -0,0 +1,14 @@
+.tags {
+  &__more-button {
```

Comment:
please extract it into reusable BEM (scss only) component and use here and in accounts component